src/autolabel/labeler.py

```diff
@@ -217,28 +223,6 @@ def run(
 
             cost += sum(response.costs)
             postfix_dict[self.COST_KEY] = f"{cost:.2f}"
-
-            # Evaluate the task every eval_every examples
-            if (current_index + 1) % eval_every == 0:
-                llm_labels = self.get_all_annotations()
-                if dataset.gt_labels:
-                    eval_result = self.task.eval(
-                        llm_labels,
-                        dataset.gt_labels[: len(llm_labels)],
-                        additional_metrics=additional_metrics,
-                    )
-
-                    for m in eval_result:
-                        # This is a row wise metric
-                        if isinstance(m.value, list):
-                            continue
-                        elif m.show_running:
```

Comment:
this gives running metrics to the labeler. Any reason for removing this? good for sanity checking